pkg/webui/console/containers/device-payload-formatters/uplink.js

```diff
@@ -65,23 +65,36 @@ class DevicePayloadFormatters extends React.PureComponent {
   static propTypes = {
     appId: PropTypes.string.isRequired,
     devId: PropTypes.string.isRequired,
-    formatters: PropTypes.formatters.isRequired,
+    formatters: PropTypes.formatters,
     updateDevice: PropTypes.func.isRequired,
   }
 
+  static defaultProps = {
+    formatters: undefined,
+  }
+
   async onSubmit(values) {
-    const { appId, devId, formatters, updateDevice } = this.props
+    const { appId, devId, formatters: initialFormatters, updateDevice } = this.props
+
+    if (values.type === PAYLOAD_FORMATTER_TYPES.DEFAULT) {
+      return updateDevice(appId, devId, {
+        formatters: null,
+      })
+    }
```

Comment:
Idem.

Reply:
This is intentional. You cannot set a particular formatter to null.